service/src/app/client.py

```
@@ -0,0 +1,116 @@
+    app.server.config["APP_STATE"] = state
+
+    app.run(debug=True)
+
+
+@callback(
+    Output(component_id="graph", component_property="figure"),
+    Output(component_id="table", component_property="rowData"),
+    Input(component_id="submit", component_property="n_clicks"),
+    State(component_id="input", component_property="value"),
+)
+def update_graph(clicks, input):
```

Ruff (0.15.7)

[error] Function argument `input` is shadowing a Python builtin

(A002)